MIPS/wallet-fee-change

```diff
@@ -0,0 +1,63 @@
+---
+eip: <to be assigned>
+title: <Reduce Mina Wallet Creation Fee>
+description: <The Proposal is to reduce the wallet creation for from 1Mina to .25Mina at the next hard fork on condition that a permanent solution is worked on.>
+author: <Peter Greystoke with help from Trivo <info@minacrypto.com>>
+discussions-to: <https://forums.minaprotocol.com/t/do-you-think-there-should-be-a-fixed-price-to-set-up-a-mina-wallet/4507/32>
+status: Draft
+type: <Standards Track>
+category (*only required for Standards Track): <Core>
+created: <2021-12-20>
+requires (*optional): <EIP number(s)>
+---
+
+MIP Template
+This is the suggested template for new MIPs.
+
+Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `wallet-fee-change.md`.
+
+The title should be 44 characters or less. It should not repeat the MIP number in title, irrespective of the category.
+
+## Abstract
+
+Reduce new wallet creation fee from 1mina to .25mina with a caveat to investigating a permenant solution through MinaResearch. 
+
+## Motivation
+
+The current 1 mina creaiton fee while still relatively small is a barrier for smaller investors. Should the price of Mina rise this will only become more so. With the prevalance of so many other coins, especially in the ZK space that don't have this fee, it may be that Mina Protocol loses out on early investors loyalty. This has additional consequences for community traction and wider interest and developer engagement. 
+
+## Specification
+
+***** HELP NEEDED TO COMPLETE THIS PART ****** 
```

Comment:
Once you have sought out the help you need, please remove this line so that it is clear that the MIP is ready to go.